### PR TITLE
update the python binding document

### PR DESCRIPTION
**Pull request by you-n-g**
RT

### CODE REVIEW

File: binding/python/multiverso/theano_ext/sharedvar.py [hunk 1]
```diff
@@ -69,5 +69,10 @@ def mv_shared(*args, **kwargs):
 
 
 def sync_all_mv_shared_vars():
+    '''
+    Sync value with multiverso. It is often used when you are training model,
+    and it will add the gradients (delta value) to the server and update the
+    latest value from the server
```
Comment (huxuan) [inline binding/python/multiverso/theano_ext/sharedvar.py]: Add a period `.` and the end of comments.

Comment (you-n-g) [inline binding/python/multiverso/theano_ext/sharedvar.py]: Amended now :)

File: binding/python/README.md [hunk 2]
```diff
@@ -45,7 +62,7 @@ self.W = theano.shared(
 )
 ```
 
-If you want to use multiverso, you can modify them like this.
+If you want to use multiverso, you can modify them like this. `mv_shared` is just a wrapper of theano.shared. It acts same as theano.shared, but it give you convenient interface to sync values.
```
Comment (huxuan) [inline binding/python/README.md]: Maybe it's better to add backticks (`) around

`theano.shared`

just like this.

Comment (you-n-g) [inline binding/python/README.md]: Amended

**Pull request merged**